onnxruntime/python/tools/transformers/models/llama/llama_parity.py

```diff
@@ -97,12 +104,14 @@ def verify_parity(args: argparse.Namespace, config: LlamaConfig, pt_model: Llama
         past_seq_len=past_sequence_length,
         max_seq_len=max_sequence_length,
         device=args.execution_provider,
-        device_id=int(args.device_id),
+        device_id=int(args.rank),
     )
 
+    logger.debug(f'ORT input: {inputs}')
```

Comment:
Can we remove this? There are a lot of inputs that will print

Reply:
sure, let me remove these